packages/rfw/lib/src/flutter/argument_decoders.dart

```diff
@@ -1420,3 +1447,57 @@ class ArgumentDecoders {
     }
   }
 }
+
+// TODO(shuoch-g): Move this class to other places where more appropriate.
+/// An implementation of [SliverPersistentHeaderDelegate] that takes in fixed
+/// child widget.
```

Comment:
```suggestion
/// An implementation of [SliverPersistentHeaderDelegate] that takes in a fixed
/// child widget.
```

Reply:
though i'm not really sure what this means exactly

Reply:
Updated the doc comment, does it help now ?